Exchange/ExchangeView/ExchangeViewModel.swift

```diff
@@ -58,59 +37,109 @@ extension ExchangeViewModel: ViewModelType {
         router: Routes
     ) -> Self {
         
-        let firstCurrencyNameInBox = Box<String>("")
-        let secondCurrencyNameInBox = Box<String>("")
-        let firstCurrencyCalculatedValue = Box<String>("")
-        let secondCurrencyCalculatedValue =  Box<String>("")
-        let networkError = Box<Error?>(nil)
-    
-        binding.didPressedSelectCurrenncyButton = { buttonNumber in
-            switch buttonNumber {
-            case .first:
+        let didReceiveError = PublishRelay<String>()
+        let firstCurrency = BehaviorRelay<String>(value: "")
+        let secondCurrency = BehaviorRelay<String>(value: "")
+        let ratesForFirstCurrency = BehaviorRelay<Dictionary<String, Double>>(value: [:])
```

Comment:
Не проще так читается?
```suggestion
        let ratesForFirstCurrency = BehaviorRelay(value: [String: Double]())
```

Reply:
исправил